.gitmodules

```diff
@@ -4,4 +4,4 @@
 	branch = v1.7.1
 [submodule "lib/suave-std"]
 	path = lib/suave-std
-	url = https://github.com/flashbots/suave-std
+	url = https://github.com/zeroXbrock/suave-std
```

Comment:
change back

Reply:
yup, will do as soon as [PR29](https://github.com/flashbots/suave-std/pull/29) is merged into the flashbots suave-std repo. The commit this project relies on exists only in my fork rn.